plugins/modules/packaging/os/yum_versionlock.py

```diff
@@ -0,0 +1,150 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# Copyright: (c) 2018, Florian Paul Hoberg <florian.hoberg@credativ.de>
+#
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+DOCUMENTATION = r'''
+---
+module: yum_versionlock
+version_added: 2.0.0
+short_description: Locks / Unlocks an installed package(s) from being updated by yum package manager.
```

Comment:
```suggestion
short_description: Locks / Unlocks an installed package(s) from being updated by yum package manager
```

Reply:
The `Unlocks` should have a lower-case `u`.

Reply:
I think you should also drop the "an", it's not applicable to the plural.

Reply:
@Andersson007 @felixfontein @onitake 
Fixed all suggestions in https://github.com/ansible-collections/community.general/pull/1405/commits/193be959fb625025d26ece397e43fbe8e0127be1.
I've also added myself as an author in https://github.com/ansible-collections/community.general/pull/1405/commits/6a3e3a5c55233d4eab9fd7a0f9219330dfed6e2e.